apps/cli/src/node/devel/docker-compose.yml

```diff
@@ -0,0 +1,150 @@
+version: "3.9"
+
+x-database-config: &database-config
+    POSTGRES_HOSTNAME: localhost
+    POSTGRES_PORT: "5432"
+    POSTGRES_USER: postgres
+    POSTGRES_PASSWORD: postgres
+    POSTGRES_DB: postgres
+
+services:
+    anvil:
+        image: sunodo/anvil
+        environment:
+            ANVIL_IP_ADDR: 0.0.0.0
+        ports:
+            - "8545:8545"
+
+    rollups_deployer:
+        image: cartesi/rollups-hardhat:devel
+        command:
+            [
+                "deploy",
+                "--network",
+                "localhost",
+                "--export",
+                "/opt/cartesi/share/deployments/localhost.json",
+            ]
+        environment:
+            RPC_URL: http://anvil:8545
+        volumes:
+            - blockchain-data:/opt/cartesi/share/deployments
+            - ./.sunodo/deployments:/app/rollups/deployments
+
+    dapp_deployer:
+        image: cartesi/rollups-cli:devel
+        restart: on-failure
+        depends_on:
+            rollups_deployer:
+                condition: service_completed_successfully
+        command:
+            [
+                "create",
+                "--rpc",
+                "http://anvil:8545",
+                "--deploymentFile",
+                "/opt/cartesi/share/deployments/localhost.json",
+                "--mnemonic",
+                "test test test test test test test test test test test junk",
+                "--templateHashFile",
+                "/opt/cartesi/share/dapp-bin/0_0/hash",
+                "--outputFile",
+                "/deployments/localhost/dapp.json",
+            ]
+        volumes:
+            - blockchain-data:/opt/cartesi/share/deployments
+            - ./.sunodo/image:/opt/cartesi/share/dapp-bin/0_0:ro
+            - ./.sunodo/deployments:/deployments
+
+    set_interval_mining:
+        image: curlimages/curl:7.84.0
+        depends_on:
+            dapp_deployer:
+                condition: service_completed_successfully
+        command:
+            [
+                "-X",
+                "POST",
+                "-s",
+                "-H",
+                "Content-Type: application/json",
+                "--data",
+                '{"id":2,"jsonrpc":"2.0","method":"evm_setIntervalMining","params":[1]}',
+                "http://anvil:8545",
+            ]
+
+    validator:
+        image: sunodo/cartesi-node:latest
+        depends_on:
+            set_interval_mining:
+                condition: service_completed_successfully
+        volumes:
+            - blockchain-data:/opt/cartesi/share/deployments:ro
+            - ./.sunodo/deployments:/deployments:ro
+            - ./.sunodo/image:/opt/cartesi/share/dapp-bin/0_0
+        ports:
+            - 8080:8080
```

Comment:
```suggestion
            - "8080"
```

same here

Reply:
Again, also may be inconvenient. User will develop his frontend pointing to the graphql endpoint.